benchexec/tools/divine.py

```diff
@@ -86,16 +96,15 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         and should give some indication of the failure reason
         (e.g., "CRASH", "OUT_OF_MEMORY", etc.).
         """
-
-        if not output:
+        if not run.output:
             return "ERROR - no output"
 
-        last = output[-1]
+        last = run.output[-1]
 
-        if isTimeout:
+        if run.was_timeout:
             return "TIMEOUT"
 
-        if returncode != 0:
+        if run.exit_code.value != 0:
             return "ERROR - Pre-run"
 
         if last is None:
```

Comment:
This is actually guaranteed to never be the case, we could remove the branch while we are at it.

Reply:
fixed in 53d68f15f868f1dca4a307f032076a0ddc5a43c3